main/settings.py

```diff
@@ -14,15 +14,23 @@
     BASE_URL = 'goadmin.ifrc.org'
 # The frontend_url nicing is in frontend.py
 
-INTERNAL_IPS = ['127.0.0.1']
+if PRODUCTION_URL is not None and 'prd' in PRODUCTION_URL:
+    DEBUG = False
+else:
+    DEBUG = True
```

Comment:
Are we setting `DEBUG=true` for all non-prod environments including staging? Enabling this might cause some security and performance issues.
cc: @batpad

Reply:
I completely agree. Saw that when going through things with @GregoryHorvath and it was unexpected - I would definitely approve of making this change to make sure DEBUG is only true on local environments.

Reply:
Right now `DEBUG=true` applies on staging as well because of the current logic, this is aimed to solve that.

Reply:
@GregoryHorvath we should definitely have DEBUG be set to False in staging as well as production.

Reply:
@thenav56 Ah sorry, I now realized I simply had a brain-fart at the moment when testing this. The previous logic worked perfectly for staging as well. I'll roll that back. cc @batpad